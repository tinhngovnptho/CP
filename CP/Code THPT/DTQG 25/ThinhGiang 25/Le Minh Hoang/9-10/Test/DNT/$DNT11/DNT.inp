68 10
0 1 1 0 1 0 0 0 0 0 1 0 1 1 1 0 0 1 0 0 1 0 1 0 1 1 1 0 1 0 0 0 0 0 1 0 1 0 0 0 0 1 1 1 0 1 1 0 0 0 0 0 1 1 0 1 1 1 0 1 0 1 0 0 1 0 1 1 
23 17 4 0 28 31 1 26 18 21 