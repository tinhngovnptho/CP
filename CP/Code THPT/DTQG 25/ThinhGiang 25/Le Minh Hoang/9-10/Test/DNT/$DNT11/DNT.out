20
80
380
517
0
0
481
2
68
35
